example/example_resourcemanager_test.go

```diff
@@ -0,0 +1,114 @@
+
+func CreateStack(ctx context.Context, provider common.ConfigurationProvider, client *resourcemanager.ResourceManagerClient) string {
+	stackName := fmt.Sprintf("test-%s", helpers.GetRandomString(8))
+	region, _ := provider.Region()
+	tenancy_ocid, _ := provider.TenancyOCID()
+
+	req := resourcemanager.CreateStackRequest{
+		CreateStackDetails: resourcemanager.CreateStackDetails{
+			CompartmentId: helpers.CompartmentID(),
+			ConfigSource: resourcemanager.CreateZipUploadConfigSourceDetails{
```

Comment:
please add some comments here showing that user can have multiple ways to create stack, might be better to add following link for others to reference [ORM Create Stack](https://docs.cloud.oracle.com/iaas/api/#/en/resourcemanager/20180917/datatypes/CreateConfigSourceDetails)